demos/workflow-cps-global-lib/README.md

```diff
@@ -1,6 +1,7 @@
 # Configure global libraries plugin
 
-Global Pipeline Libraries plugin configuration belongs under `unclassified` root element
+Global Pipeline Libraries plugin configuration belongs under the `unclassified` root element.
+The [`configuration-as-code-support`] plugin needs to be installed as well.
```

Comment:
@ewelinawilkosz Until the changes have been upstreamed could the plugin just depend on configuration-as-code-support.
For the majority of people I assume they need the support plugin, if you're touching credentials, users, jobs etc.

Not sure if it is actually needed in this case though